.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,63 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test-build-release:
+    env:
+      IS_PUSH_TO_MASTER: ${{ github.event.push && github.ref == 'refs/heads/main' }}
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Install npm packages using cache
+        uses: bahmutov/npm-install@v1
+      - name: Lint code
+        run: yarn lint
+      - name: Run tests with coverage
+        run: yarn cover
+
+      # at this point, the build is successful
+      - name: Codecov
+        uses: codecov/codecov-action@v1.2.0
+        continue-on-error: true
+      - name: Semantic Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: yarn semantic-release
+
+      # Steps below deal with uploading Storybook to S3,
+      # which is only performed with a push to the main branch.
+      - name: Add AWS credentials
+        if: ${{ env.IS_PUSH_TO_MASTER }}
+        run: mkdir ~/.aws && printf '%s\n' '[default]' "aws_access_key_id=$AWS_ACCESS_KEY_ID" "aws_secret_access_key=$AWS_SECRET_ACCESS_KEY" "region=$AWS_REGION" > ~/.aws/config
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_REGION: ${{ secrets.AWS_REGION }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      # Install AWS CLI v1 using pip for uploading storybook to S3
+      # (AWS CLI v2 does not come with a non-sudo installer as of writing).
+      - name: Set up Python (latest 3.x)
+        if: ${{ env.IS_PUSH_TO_MASTER }}
+        uses: actions/setup-python@v2
+      - name: Install pip
+        if: ${{ env.IS_PUSH_TO_MASTER }}
+        run: pip install -U pip
+      - name: Install AWS CLI v1
+        if: ${{ env.IS_PUSH_TO_MASTER }}
+        run: pip install awscli
```

Comment:
Looks like the AWS CLI is already installed on Ubuntu: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#cli-tools

So, we shouldn't need this step or the ones prior (python install, pip install).

Reply:
Good find, and it looks like AWS CLI v2 is preinstalled on there, so it saves on execution time (see https://github.com/conveyal/transitive.js/pull/63/commits/2a826d6ead65a21adf2ebb6990814ee2b8bdb44b)